oraichain/assetlist.json

```diff
@@ -50,6 +50,526 @@
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "orai10ldgzued6zjp0mkqwsv2mux3ml50l97c74x8sg",
+      "base": "cw20:orai10ldgzued6zjp0mkqwsv2mux3ml50l97c74x8sg",
+      "name": "Airight",
+      "display": "airi",
+      "coingecko_id": "airight",
+      "symbol": "AIRI",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/airi.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/airi.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/airi.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/airi.svg"
+      }
+    },
+    {
+      "description": "The ORAIX token is the cw20 token for the Oraichain.",
+      "denom_units": [
+        {
+          "denom": "cw20:orai1lus0f0rhx8s03gdllx2n6vhkmf0536dv57wfge",
+          "exponent": 0
+        },
+        {
+          "denom": "oraix",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "orai1lus0f0rhx8s03gdllx2n6vhkmf0536dv57wfge",
+      "base": "cw20:orai1lus0f0rhx8s03gdllx2n6vhkmf0536dv57wfge",
+      "name": "Oraix",
+      "display": "oraix",
+      "symbol": "ORAIX",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/oraix.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/oraix.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/oraix.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/oraix.svg"
+      }
+    },
+    {
+      "description": "USDC issued on Oraichain.",
+      "denom_units": [
+        {
+          "denom": "cw20:orai15un8msx3n5zf9ahlxmfeqd2kwa5wm0nrpxer304m9nd5q6qq0g6sku5pdd",
+          "exponent": 0
+        },
+        {
+          "denom": "usdc",
+          "exponent": 6
+        }
+      ],
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "noble-1",
+            "base_denom": "uusdc",
+            "channel_id": "channel-34"
+          },
+          "chain": {
+            "channel_id": "channel-301",
+            "path": "transfer/channel-301/uusdc"
+          }
+        }
+      ],
+      "type_asset": "cw20",
+      "address": "orai15un8msx3n5zf9ahlxmfeqd2kwa5wm0nrpxer304m9nd5q6qq0g6sku5pdd",
+      "base": "cw20:orai15un8msx3n5zf9ahlxmfeqd2kwa5wm0nrpxer304m9nd5q6qq0g6sku5pdd",
+      "name": "Usdc",
+      "display": "usdc",
+      "symbol": "USDC",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/axelar/images/usdc.png"
```

Comment:
Please use the _non-cosmos/ethereum USDC image instead (this Axelar image may soon be deleted).
And please remove your upload of USDC.